--- origsrc/mypaint-1.1.0/brushlib/fifo.c	2012-12-31 11:03:38.000000000 -0600
+++ src/mypaint-1.1.0/brushlib/fifo.c	2013-01-20 03:43:32.275919700 -0600
@@ -15,7 +15,7 @@
  *
  */
 
-#include <malloc.h>
+#include <stdlib.h>
 #include "fifo.h"
 
 struct fifo_item
--- origsrc/mypaint-1.1.0/brushlib/operationqueue.c	2012-12-31 11:03:38.000000000 -0600
+++ src/mypaint-1.1.0/brushlib/operationqueue.c	2013-01-20 03:44:23.657444400 -0600
@@ -15,7 +15,7 @@
  */
 
 
-#include <malloc.h>
+#include <stdlib.h>
 #include <assert.h>
 
 #include <mypaint-glib-compat.h>
--- origsrc/mypaint-1.1.0/brushlib/tests/SConscript	2012-12-31 11:03:38.000000000 -0600
+++ src/mypaint-1.1.0/brushlib/tests/SConscript	2013-01-20 03:48:46.382806200 -0600
@@ -1,6 +1,6 @@
 Import('env', 'gegl_env')
 
-import os
+import os, sys
 
 testlib_env = env.Clone()
 
@@ -28,6 +28,8 @@ tests_env = testlib_env.Clone()
 # Build individual tests
 tests_env['LIBS'].remove('mypaint') # Workaround: ordering wrong when using Prepend on some systems
 tests_env.Prepend(LIBS=['mypaint-tests', 'mypaint'])
+if sys.platform == 'cygwin':
+    tests_env.Append(LIBS=['-lintl'])
 
 # Standard tests
 for source in tests_sources:
